client/state/selectors/test/get-current-user-payment-methods.js

```diff
@@ -176,18 +93,35 @@ describe( 'getCurrentUserPaymentMethods()', () => {
 	test( 'fr-FR should return credit card primary, PayPal secondary', () => {
 		update( 'geo', 'success', 'FR' );
 
-		expect( getCurrentUserPaymentMethods( frLangFRCountryState ) ).to.eql( [
+		expect( getCurrentUserPaymentMethods( countryStateFromLocale('fr') ) ).to.eql( [
 			'credit-card',
 			'paypal',
 		] );
 	} );
 
 	test( 'BR should return credit card primary, tef, and PayPal secondary', () => {
 		update( 'geo', 'success', 'BR' );
-		expect( getCurrentUserPaymentMethods( ptBrBRCountryState ) ).to.eql( [
+
+		let countryState = countryStateFromLocale('pt-br');
```

Comment:
Note the [circleci message ](https://circleci.com/gh/Automattic/wp-calypso/101260?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)about this line:
```
/home/circleci/wp-calypso/client/state/selectors/test/get-current-user-payment-methods.js
  105:7  error  'countryState' is never reassigned. Use 'const' instead  prefer-const
```